docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,73 @@
+First, to ensure you install Endpoint on the desired k8s cluster(s), use `kubectl config use-context <name-of-context>` to set the default context. You can repeat the following process for multiple contexts.
+
+NOTE: To check which contexts exist, use `kubectl config get-contexts` to list them from your local kubectl config file. An asterisk, if present, indicates the current default context.
+
+To apply the manifest to a cluster, run for example: `kubectl apply -f eks-elastic-security.yaml` for EKS, or `kubectl apply -f gke-elastic-security.yaml` for GKE.
+
+To check the daemonset’s status, run for example: `kubectl get pods -A`
+
+Once the daemonset is running, Elastic Endpoint will start sending Linux session data from Kubernetes to Kibana. You can then view that data using the Kubernetes dashboard.
+
+[discrete]
+== The Kubernetes dashboard
+To find the dashboard, go to **Security > Dashboards > Kubernetes**.
+
+image::kubernetes-dashboard.png[the Kubernetes dashboard, with numbered labels 1 through 3 for major sections]
+Sections numbered above are described below:
+
+  1. The charts at the top of the dashboard provide an overview of your monitored Kubernetes infrastructure. You can hide them by clicking Hide charts.
+  2. The Tree navigation menu allows you to navigate through your deployments and select the scope of the Sessions table to the right. You can select any item in the menu to show its sessions. In Logical view, the menu is organized by Cluster, Namespace, Pod, and Container image. In Infrastructure view, it is organized by Cluster, Node, Pod, and Container image.
+  3. The sessions table displays sessions collected from the selected element of your Kubernetes infrastructure. You can view it in fullscreen by selecting the button in its upper right corner, and sort it by any of its fields.
+
+You can filter the data using the KQL search bar and date picker at the top of the page, and take the following investigative actions on sessions in the sessions table using the buttons to their left:
+
+- View details
+- Open in {security-guide}/timelines-ui.html[timeline]
+- Run {kibana-ref}/osquery.html[Osquery]
+- {security-guide}/visual-event-analyzer.html[Analyze] event
+- Open {security-guide}/session-view.html[Session View]
+
+IMPORTANT: This dashboard uses data from the `logs-*` index pattern in the {security-guide}/advanced-settings.html#update-sec-indices[`securitySolution:defaultIndex` advanced setting] (where it is present by default). To collect data from multiple {es} clusters (as in a cross-cluster deployment), update `logs-*` to `*:logs-*` (this setting determines which data appears throughout Elastic Security).
+
+== Kubernetes-specific data in Session view
+
+Session view displays Kubernetes-specific metadata under the Metadata tab of the Detail panel:
+
+image::metadata-tab.png[the Detail panel's metadata tab]
+
+The metadata tab is organized into four expandable sections:
```

Comment:
```suggestion
The *Metadata* tab is organized into four expandable sections:
```

Reply:
I'd also omit "four" because it's just one more word you have to remember to revise if they ever add/remove sections:
```suggestion
The *Metadata* tab is organized into these expandable sections:
```